index.html

```diff
@@ -0,0 +1,235 @@
+
+  <!-- MAIN CONTAINER -->
+  <div class="topBarContainer">
+    <div class="container">
+      <div class="topBar">
+        <a href="https://twitter.com/magic_breakfas" title="Magic Breakfast Twitter"><i class="fab fa-twitter"></i></a>
+        <a href="https://www.facebook.com/magicbreakfastuk/?fref=ts" title="Magic Breakfast Facebook"><i class="fab fa-facebook"></i></a>
+        <a href="https://www.youtube.com/user/MagicBreakfastUK" title="Magic Breakfast You Tube"><i class="fab fa-youtube"></i></a>
+        <a href="https://www.instagram.com/magicbreky/" title="Magic Breakfast Instagram"><i class="fab fa-instagram"></i></a>
+        <a href="/Lobby/SignUp.aspx?ReturnURL=%2fDefault.aspx" title="Become a registered member of our platform">LOG
+          IN</a>
+        <a href="#" class="menuAdminSearchLink"><i class="fas fa-search"></i></a>
+      </div>
+    </div> <!-- end of con-->
+  </div>
+  <!-- END OF TOP BAR CONTAINER-->
+
+  <div class="navBarContainer">
+    <div class="container">
+      <div class="navBar">
+        <div class="navBar-button">HOME</div>
+        <div class="navBar-content navBar-hidden">
+            <a class="dropdown-link" href="https://lambdaschool.com/" target="_blank">ABOUT</a>
+            <a class="dropdown-link" href="https://www.google.com/" target="_blank">GET INVOLVED</a>
+            <a class="dropdown-link" href="https://developer.mozilla.org/en-US/" target="_blank">SCHOOLS</a>
+          </div>
+        </div>
+       <!-- work on this later
+        <a href="#" title="HOME">HOME</a>
+        <a href="#" title="ABOUT">ABOUT</a>
+        <a href="#" title="GET INVOLVED">GET INVOLVED</a>
+        <a href="#" title="SCHOOLS">SCHOOLS</a>
+        <a href="#" class="donate" title="DONATE">DONATE</a>
+      </div> -->
+    </div> <!-- end of con-->
+  </div>
+  <!-- END OF NAV BAR CONTAINER-->
+
+  <div class="jumbotronContainer">
+    <img class="image1" src="assets/pic1.jpeg" alt="Let's go on an adventure!">
+    <div class="sideDetail">NO CHILD TOO</br> HUNGRY TO LEARN</div>
+  </div>
+  <!-- END OF JUMBOTRON CONTAINER-->
+
+  <div class="midContainer1">
+    <div class="container">
+      <div class="midText">
+        <p>WELCOME TO MAGIC BREAKFAST. WE ARE A REGISTERED CHARITY AIMING TO END HUNGER AS A BARRIER TO</p>
+        EDUCATION IN UK SCHOOLS THROUGH THE PROVISION OF HEALTHY BREAKFASTS TO VULNERABLE CHILDREN
+      </div>
+    </div> <!-- end of con-->
+  </div>
+  <!-- END OF FIRST MID CONTAINER -->
+
+  <div class="midContainer2">
+    <div class="boxItems5">
+      <img class="items6" src="assets/image2.jpeg">
+      <img class="items6" src="assets/image3.jpeg">
+      <img class="items6" src="assets/image4.jpeg">
+    </div>
+  </div>
+  <!-- END OF SECOND MID CONTAINER -->
+
+  <div class="midContainer3">
+    <div class="container">
+      <div class="boxItems6">
+        <img class="items7" src="assets/image5.png">
```

Comment:
be more descriptive with classes. if you have similar images, use reuse the same class